DSC/dsc.py

```diff
@@ -47,6 +49,8 @@
 dsc_build = 1
 dsc_release = 294
 package_pattern = '(\d+).(\d+).(\d+).(\d+)'
+nodeid_path = '/etc/opt/omi/conf/dsc/agentid'
+DateTimeFormat = "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
Could you use the snake notation here too (with _ as a separator, and lowercasing)